src/ContryAndCity.py

```diff
@@ -0,0 +1,25 @@
+# Задача Дан список стран и городов каждой страны.
+# Затем даны названия городов.
+# Для каждого города укажите, в какой стране он находится.
+NumberOfCountries = int(input("enter the number of countries"))
+count = 0
+data = {}
+
+while count != NumberOfCountries:
+    count += 1
+    CountriesAndCities = str(input("enter countries and cities"))
+    CountriesAndCities = CountriesAndCities.split()
+    data[CountriesAndCities[0]] = CountriesAndCities[1:]
+
+NumberOfRequests = int(input("enter the number of requests"))
+count2 = 0
+DataCities = []
+
+while count2 != NumberOfRequests:
+    count2 += 1
+    DataCities.append(input("Enter cities"))
+
+for city in DataCities:
+    for country, town in data.items():
```

Comment:
O(n*m) complexity in fact. it seems it would much better if we'll create dict like city: country.